debug/Makefile

```diff
@@ -11,19 +12,27 @@ all-tests: spike32 spike32-2 spike32-2-rtos spike32-2-hwthread \
 
 all:	pylint all-tests
 
-# Target to check all the multicore options.
-multi-tests: spike32-2 spike64-2-rtos spike32-2-hwthread
-
-pylint:
-	pylint --rcfile=pylint.rc `git ls-files '*.py'`
+foo:
```

Comment:
Just want to double check the intention was to keep "foo" as the name of that particular make target, or whether that was a placeholder name or temporary scaffolding (not that it really matters, but figured I'd ask).

Reply:
Oh, that was just a test while making this work. Thanks for noticing. I'll remove it.